docs/examples/changing-the-src-dir.md

```diff
@@ -0,0 +1,100 @@
+---
+description: |
+  Change the source directory to effectively manage your project.
+---
+
+When you initialize a project with `deno run -A -r https://fresh.deno.dev`,
+you'll end up with a project like the following:
+
+```
+.
+├── README.md
+├── components
+│   └── Button.tsx
+├── deno.json
+├── dev.ts
+├── fresh.gen.ts
+├── islands
+│   └── Counter.tsx
+├── main.ts
+├── routes
+│   ├── [name].tsx
+│   ├── api
+│   │   └── joke.ts
+│   └── index.tsx
+└── static
+    ├── favicon.ico
+    └── logo.svg
+```
+
+## Using a `src` directory
+
+If you'd like your code to live in a `src` directory (or any other place of your
+choosing), then you'll need to do the following things:
+
+1. Move most of the code to your chosen directory. For this example I'll stick
+   with `src`.
```

Comment:
```suggestion
1. Move all your files, except `deno.json` and `README.md`, to the `src` directory.
```
For greater clarity. Also, best not to have written in a first-person point of view in the documentation.

Reply:
yes, sounds better